Releases.md

```diff
@@ -2,24 +2,24 @@
+| [1.6.0](#directml-160)   | [DML_FEATURE_LEVEL_4_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_4_0) | Windows 11 (Build 10.0.22000; 21H2)                               | [Microsoft.AI.DirectML.1.6.0](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.6.0)   |
+| [1.5.1](#directml-151)   | [DML_FEATURE_LEVEL_3_1](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_3_1) | Redistributable only                                              | [Microsoft.AI.DirectML.1.5.1](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.5.1)   |
+| [1.5.0](#directml-150)   | [DML_FEATURE_LEVEL_3_1](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_3_1) | Redistributable only                                              | [Microsoft.AI.DirectML.1.5.0](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.5.0)   |
+| [1.4.3](#directml-143)   | [DML_FEATURE_LEVEL_3_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_3_0) | Redistributable only                                              | [Microsoft.AI.DirectML.1.4.3](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.4.3)   |
+| [1.4.2](#directml-142)   | [DML_FEATURE_LEVEL_3_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_3_0) | Redistributable only                                              | [Microsoft.AI.DirectML.1.4.2](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.4.2)   |
+| [1.4.1](#directml-141)   | [DML_FEATURE_LEVEL_3_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_3_0) | Redistributable only                                              | [Microsoft.AI.DirectML.1.4.1](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.4.1)   |
+| [1.4.0](#directml-140)   | [DML_FEATURE_LEVEL_3_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_3_0) | Redistributable only                                              | [Microsoft.AI.DirectML.1.4.0](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.4.0)   |
+| [1.1.0](#directml-110)   | [DML_FEATURE_LEVEL_2_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_2_0) | Windows 10 May 2020 Update, Version 2004 (Build 10.0.19041, 20H1) | -                                                                                           |
+| [1.0.0](#directml-100)   | [DML_FEATURE_LEVEL_1_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_1_0) | Version 1903 (Build 10.0.18362; 19H1, "May 2019 Update")          | -                                                                                           |
```

Comment:
Notice these two lines are inconsistent now. Can you make them harmonious?

`Windows 10 May 2020 Update, Version 2004 (Build 10.0.19041, 20H1)`
`Version 1903 (Build 10.0.18362; 19H1, "May 2019 Update")`